src/plugins/foxPage/hooks/useOtherOpportunities.ts

```diff
@@ -37,7 +37,6 @@ export const useOtherOpportunities = (assetId: AssetId) => {
                       .plus(lpApr ?? 0)
                       .toString()
                   : null,
-              link: 'https://fox.shapeshift.com/fox-farming/liquidity/0x470e8de2ebaef52014a47cb5e6af86884947f08c/staking/0x24fd7fb95dc742e23dc3829d3e656feeb5f67fa0/get-started',
```

Comment:
what do here?

Reply:
Since we now have FOX/ETH LP and Farming in web, this will now use the modal, or route to "Connect Wallet" depending on the connected wallet:

#### Keplr
<img width="819" alt="image" src="https://user-images.githubusercontent.com/17035424/194132283-876f3907-8997-439c-8097-ac364f870949.png">

#### Native/MM/ any other wallet supporting ETH

<img width="831" alt="image" src="https://user-images.githubusercontent.com/17035424/194132390-1ef0bcce-3ebf-48eb-b5fd-05bb0b4fb6f4.png">